### ISSUE TITLE
Input Boolean Fields Not Displaying On iOS App

### ISSUE THREAD

**Issue by proxlx** [home-assistant/frontend#11574]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The input Boolean fields are not displaying correctly on the HA iOS app. They show up fine when using a browser such as Chrome. 

iOS App View

![39409D18-4672-4626-A9F5-EE2E688A2165](https://user-images.githubusercontent.com/64808492/152680032-69901120-773f-4082-9cbc-1988d6056f75.jpeg)

Browser View

![45B6E4AC-0644-4D7C-8781-F7E51BE18920](https://user-images.githubusercontent.com/64808492/152680217-e62664ba-c53d-469a-9003-91d837a406e4.jpeg)

### Describe the behavior you expected

You should be able to view the field as previously was the case. You can still type and it takes the input but cannot see the digits

### Steps to reproduce the issue

1. Add Input Boolean To Dashboard
2. View dashboard on iOS app - does not display digits
3. View on browser - displays ok

### What version of Home Assistant Core has the issue?

2022.2.2

### In which browser are you experiencing the issue with?

Ios Application 

### Which operating system are you using to run this browser?

IOS 15.2

### Additional information

I thought this maybe linked to the recent Gauge Cards not displaying properly but after updating to 2022.2.2 they now work fine but the input Booleans still don’t display correctly

**Comment by emufan** [comment]
Is it really boolean and not date_time, you are talking about? If the white boxes in date time, then perhaps the same as https://github.com/home-assistant/frontend/issues/10499?

**Comment by proxlx** [comment]
@emufan You are quite correct! That’s my fault looking at the wrong entity in the card!! Yes the issue lies with input_date time and is the same as the issue you mentioned!

**Comment by emufan** [comment]
As we are currently not that many with the problems and nothings happened with my links issue till now, would you upvote linked one and add that you would like to get this fixed as well?

**Comment by emufan** [comment]
If a frontend developer will have a look in this, the problem seems to be related to iron input. Tested around with card_mod

```
    hui-input-datetime-entity-row$:
      ha-time-input$:
        paper-time-input$:
          paper-input:
            $: |
              iron-input { background: red; };
```

WIll show red in chrome, whereas it is still white in webkit,

Wheras

```
card_mod:
  style:
    hui-input-datetime-entity-row$:
      ha-time-input$:
        paper-time-input$:
          paper-input:
            $: |
              iron-input input { background: red; };
```

shows up in red in both, but in Webkit rounded and still with the white paper-input. So for now, I removed the radius there as well and have a workaround.

But the problem, that the paper-input element always shows white background, is still there. Or black in HA-dark-theme setting. It is neither inheriting the parent backgrounds nor anything else (in webkit).

So it seems to be still a webkit bug in combination with iron input, but should be still solved.
